### PR TITLE
[Backport release-3_0] Refine manual log submission

### PR DESCRIPTION
**Pull request by qfield-fairy**
Backport #4710
 **Authored by:** @nirvn

### CODE REVIEW

**Comment by qfield-fairy** [comment]
🎉 Ta-daaa, freshly created APKs are available for c8be8b093cd76800884fc01488483a04f9a9a8f6: [**arm64-android**](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-4714-230a934220fb5ae628fd76059f15e0484bd166a5-arm64-android.apk)
<details>
<summary>Other architectures</summary>

- [arm-neon-android](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-4714-230a934220fb5ae628fd76059f15e0484bd166a5-arm-neon-android.apk)
- [x64-android](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-4714-230a934220fb5ae628fd76059f15e0484bd166a5-x64-android.apk)
- [x86-android](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-4714-230a934220fb5ae628fd76059f15e0484bd166a5-x86-android.apk)
- [**all access arm64-android**](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield_all_access-dev-4714-230a934220fb5ae628fd76059f15e0484bd166a5-arm64-android.apk)
- [all access arm-neon-android](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield_all_access-dev-4714-230a934220fb5ae628fd76059f15e0484bd166a5-arm-neon-android.apk)
- [all access x64-android](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield_all_access-dev-4714-230a934220fb5ae628fd76059f15e0484bd166a5-x64-android.apk)
- [all access x86-android](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield_all_access-dev-4714-230a934220fb5ae628fd76059f15e0484bd166a5-x86-android.apk)
</details>

File: src/qml/MessageLog.qml [hunk 1]
```diff
@@ -143,16 +143,78 @@ Page {
...
+        indicator.implicitHeight: 24
+        indicator.implicitWidth: 24
+      }
+    }
+
+    standardButtons: Dialog.Ok | Dialog.Cancel
+    onAccepted: {
+      var applicationLogMessage = appliationLogInput.text.trim()
+      iface.sendLog(applicationLogMessage != '' ? applicationLogMessage : 'Manual log submission', includeCloudInformationCheckBox.checked ? cloudConnection.username : '')
+      displayToast(qsTr("Your application log is being sent"))
```
Comment (suricactus) [inline src/qml/MessageLog.qml:214]:
```suggestion
      displayToast(qsTr("Your application log is being sent…"))
```
It's good sign that is WIP and not a finished action.

**Review by suricactus — APPROVED** [review]
Good move! Sorry for putting comments in the backport PR, tiny improvement suggestions.

File: src/qml/MessageLog.qml (hunk 1 above)
Comment (nirvn) [reply src/qml/MessageLog.qml:214]: @suricactus , I'll change it on master but not on release-3_0 to avoid breaking a translation string. Good catch!

**Pull request merged**